library/src/main/kotlin/com/vinted/actioncable/client/kotlin/Consumer.kt

```diff
@@ -77,6 +78,10 @@ class Consumer(
         }
     }
 
+    fun reopen() {
```

Comment:
In what kind of use case you would expect to call this one?

At the moment, ```consumer.reopen()``` won't work if it is invoked after ```consumer.disconnect()``` call because ```EventsHandler``` coroutine context is cancelled.

Reply:
i use reopen fun when connect is opened.

Reply:
> i use reopen fun when connect is opened.

Did you mean you want `close` and `open` connections that already is opened?

Reply:
> > i use reopen fun when connect is opened.
> 
> Did you mean you want `close` and `open` connections that already is opened?

yes